### PR TITLE
Make Tadpole land if the console is killed mid-air

### PR DESCRIPTION
**Pull request by Wayland-Smithy**
## About The Pull Request

Title.

Makes the console slashing messages to the xeno a little clearer and follow the collective centric notice conventions.

## Why It's Good For The Game

Currently the ship stays in brazil if this happens. That is bad right?

## Changelog
:cl:
fix: Fixed Tadpole getting stuck in the air with a slashed console.
/:cl:

### CODE REVIEW

**Comment by Wayland-Smithy** [comment]
Speedmerge

File: code/modules/shuttle/mini_dropship.dm [hunk 1]
```diff
@@ -169,6 +169,14 @@
 	s2.start()
 	damaged = TRUE
 	remove_eye_control(ui_user)
+	if(fly_state == SHUTTLE_IN_ATMOSPHERE && last_valid_ground_port)
```
Comment (BraveMole) [inline code/modules/shuttle/mini_dropship.dm:172]: check for destination as well (should be fly_state), if you are already going up on ship you don't want to go on ground

Comment (Wayland-Smithy) [reply code/modules/shuttle/mini_dropship.dm:172]: I didn't really think about it, but after testing it is intended.

**Comment by Wayland-Smithy** [comment]
Speedmerge

**Comment by AnCopper** [comment]
Make it crash into a random part of the map 😎

**Comment by jroinc** [comment]
+1, very good idea.

File: code/modules/shuttle/mini_dropship.dm [hunk 2]
```diff
@@ -152,12 +152,12 @@
 	if(damaged)
 		return
 	X.visible_message("[X] begins to slash delicately at the computer",
-	"You start slashing delicately at the computer.")
+	"We start slashing delicately at the computer. This will take a while.")
```
Comment (SplinterGP) [inline code/modules/shuttle/mini_dropship.dm:155]: just for next time, we are not much going to keep the "we" stuff and all, the stuff can be targeted at the player just fine, we decided that some time ago

Comment (Wayland-Smithy) [reply code/modules/shuttle/mini_dropship.dm:155]: okay

**Review by SplinterGP — APPROVED** [review]

**Comment by SplinterGP** [comment]
seems okay to me

**Pull request merged**